pkg/state/type.go

```diff
@@ -102,3 +103,13 @@ func (r *routeState) UnmarshalJSON(b []byte) error {
 	r.routeFields = fields
 	return nil
 }
+
+type bridgeMappings struct {
+	PhysicalNetworkMappings []PhysicalNetworks `json:"bridge-mappings,omitempty" yaml:"bridge-mappings,omitempty"`
+}
+
+type PhysicalNetworks struct {
+	Name   string `json:"localnet" yaml:"localnet"`
+	Bridge string `json:"bridge" yaml:"bridge"`
+	State  string `json:"state,omitempty" yaml:"state,omitempty"`
+}
```

Comment:
This is not really needed since we don't need to filter it out, we just need an "ovn" field at desiredState level of type `json.RawMessage`

Reply:
It's not only filtering - it makes testing a lot easier, since I can use these structs to assert against. 

I prefer to 
```go
Expect(nodeBridgeMappings(node)).To(
				ConsistOf(state.PhysicalNetworks{Name: networkName, Bridge: bridgeName}))
```

instead of comparing against a string. 

But I won't insist.

Reply:
We will consume a generated golang api in the furure so I am ok with this.